File: Php/app/Urls/PlayerInviteUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\PlayerUrlModel;
	use entities\Player;
	use entities\Homegame;

	class PlayerInviteUrlModel extends PlayerUrlModel{

		public function __construct(Homegame $homegame, Player $player){
			parent::__construct(RouteFormats::playerInvite, $homegame, $player);
		}

	}

}

File: Web/Models/UrlModels/PlayerInviteUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class PlayerInviteUrlModel : PlayerUrlModel{

		public PlayerInviteUrlModel(Homegame homegame, Player player) : base(RouteFormats.PlayerInvite, homegame, player)
        {
		}

	}

}